src/test/groovy/pl/touk/repository/InMemoryDriverRepositorySec.groovy

```diff
@@ -0,0 +1,34 @@
+package pl.touk.service
+
+import pl.touk.repository.InMemoryDriverRepository
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import java.time.LocalDateTime
+import java.time.ZoneId
+
+class InMemoryDriverRepositorySec extends Specification {
+
+    @Unroll
+    def "should check two dates is same: '#execNr'"() {
+        given:
+        def first = LocalDateTime.ofInstant((new Date(year1, month1, day1)).toInstant(), ZoneId.systemDefault());
+        def second = LocalDateTime.ofInstant((new Date(year2, month2, day2)).toInstant(), ZoneId.systemDefault());
+
+        when:
+        def isSameDay = InMemoryDriverRepository.isSameDay(first, second)
```

Comment:
Dlaczego nie testujesz innych metod z InMemoryDriverRepository?

Reply:
Nie wyrobiłem się z czasem, a nie chciałem bardziej przeciągać czasu który miałem na to zadanie.